base/docker-compose.yml

```diff
@@ -42,49 +48,72 @@ services:
       - /etc/localtime:/etc/localtime:ro
       - ${ROOT}/downloads/torrent-blackhole:/downloads # place where to put .torrent files for manual download
       - ${ROOT}/config/jackett:/config # config files
-
+    labels:
+      - "traefik.backend=jackett"
+      - "traefik.local.frontend.rule=Host:jackett.localhost"
```

Comment:
does this work fine if I access an external machine on my local network? eg. `jackett.192.168.1.12`? Or is there a restriction to the `localhost` domain?

Reply:
Looking like this doesn't work, and I don't think subdomains support raw ip access in general.

I considered using subpaths instead of subdomains, however this causes issues for the default configuration of most of the apps, and I think some of them don't even support remapping the path, so subdomains work a lot better.

This seems like a good reason to keep around exposing the ports to the host so you can still access the local version from other clients.

Reply:
Alright, I've added back the ports instead of expose, so you can still access services with `localhost/ip:port`.

Reply:
I think subpath would work if combined with `PathPrefixStrip`?
What I like about the subpaths instead of subdomains solutions is that it's still possible to `curl 192.168.1.12/radarr` (no need for the host ports mapping).
What do you think?

Reply:
Looks like PathPrefixStrip isn't working as expected, sadly. I don't think it will work properly.